internal/orchestrator/updates.go

```diff
@@ -124,16 +130,33 @@ func parseStatusKVKey(key string) (*statusKey, error) {
 type flasherStatus struct {
 	UpdatedAt  time.Time       `json:"updated"`
 	WorkerID   string          `json:"worker"`
+	TraceID    string          `json:"traceID"`
+	SpanID     string          `json:"spanID"`
 	Target     string          `json:"target"`
 	State      string          `json:"state"`
 	Status     json.RawMessage `json:"status"`
 	MsgVersion int32           `json:"msgVersion"`
 }
 
+const (
+	Version int32 = 1
+)
+
+// MustBytes sets the version field of the StatusValue so any callers don't have
+// to deal with it. It will panic if we cannot serialize to JSON for some reason.
+func (v *flasherStatus) MustBytes() []byte {
```

Comment:
Do we need to publish a Flasher status from ConditionOrc?

Reply:
not really, this was only added to cut the dependency on flasher in updates_test.go

Reply:
I'm looking forward to moving all of these shared types into one place, then we have less duplication

Reply:
For the test, I'd rather actually explicitly rely on flasher (to avoid problems with flasher drifting and conditionorc being silently broken). It's only the test, and it's pretty manageable until we break that common eventing stuff out.